gp-templates/helper-functions.php

```diff
@@ -366,12 +369,17 @@ function textareas( $entry, $permissions, $index = 0 ) {
 
 function display_status( $status ) {
 	$status_labels = array(
-		'current'  => _x( 'current', 'Single Status', 'glotpress' ),
-		'waiting'  => _x( 'waiting', 'Single Status', 'glotpress' ),
-		'fuzzy'    => _x( 'fuzzy', 'Single Status', 'glotpress' ),
-		'old'      => _x( 'old', 'Single Status', 'glotpress' ),
-		'rejected' => _x( 'rejected', 'Single Status', 'glotpress' ),
+		'current'           => _x( 'current', 'Single Status', 'glotpress' ),
+		'waiting'           => _x( 'waiting', 'Single Status', 'glotpress' ),
+		'fuzzy'             => _x( 'fuzzy', 'Single Status', 'glotpress' ),
+		'old'               => _x( 'old', 'Single Status', 'glotpress' ),
+		'rejected'          => _x( 'rejected', 'Single Status', 'glotpress' ),
+		'changes_requested' => _x( 'Changes requested', 'Single Status', 'glotpress' ),
 	);
+	// If a changes_requested status exists in the database but they are no longer enabled, they will show as rejected.
+	if ( ! apply_filters( 'gp_enable_changes_requested_status', false ) ) {// todo: delete when we merge the gp-translation-helpers in GlotPress
+		$status_labels['changes_requested'] = _x( 'untranslated', 'Single Status', 'glotpress' );
```

Comment:
Comment and code are not consistent here. untranslated or rejected?

Reply:
Good catch! The status was incorrect. Solved here https://github.com/GlotPress/GlotPress/pull/1451/commits/5e286f3db15f581ac68cd0023a7a0513912b8be8.